src/Markup/Avalonia.Markup.Xaml/MarkupExtensions/StyleIncludeExtension.cs

```diff
@@ -1,22 +0,0 @@
-using Avalonia.Markup.Xaml.Styling;
-using Avalonia.Styling;
-using System.ComponentModel;
-using System;
-
-namespace Avalonia.Markup.Xaml.MarkupExtensions
-{
-    public class StyleIncludeExtension
```

Comment:
I believe this class was needed in pre-XamlX era. Now compiled can resolve normal StyleInclude just fine.